packages/minecraft.rb

```diff
@@ -12,7 +12,8 @@ class Minecraft < Package
   source_sha256 'SKIP'
 
   depends_on 'libx11'
-  depends_on 'jdk8'
+  depends_on 'openjdk8'
+  depends_on 'openjdk17'
```

Comment:
We only need one or the other.  These packages are mutually exclusive of each other.

Reply:
Hmm. That's a problem then - see the opening comment on this issue:

> Needed to run MC versions >= 1.17, see https://github.com/MultiMC/Launcher/wiki/Using-the-right-Java for an overview

Just to clarify, versions older than 1.17 _will still need Java 8_. Worst case scenario I say switch to 17 anyways and ditch the old versions, but being able to support both would be ideal...

Reply:
This Minecraft version requires 17 so I say remove the `depends_on 'openjdk8'` line.

Reply:
I can live with that, but you're 100% sure there's no way multiple versions of Java can be supported at once, like on other Linux distros?

Reply:
Not unless the executables have different names but that is not easy to do.  Are you 100% sure this Mindcraft release requires jdk8?

Reply:
Yeah, modern versions of Minecraft (and all future versions going forward) require JDK 17 and will error out or something if they're launched with version 8.